internal/pkg/service/appsproxy/config/config.go

```diff
@@ -20,6 +21,7 @@ type Config struct {
 	DNSServer        string            `configKey:"dnsServer" configUsage:"DNS server for proxy. If empty, the /etc/resolv.conf is used."`
 	API              API               `configKey:"api"`
 	CookieSecretSalt string            `configKey:"cookieSecretSalt" configUsage:"Cookie secret needed by OAuth 2 Proxy." validate:"required" sensitive:"true"`
+	AppTimeouts      Timeouts          `configKey:"appTimeouts" configUsage:"Timeouts set for application interactions"`
```

Comment:
Naming :thinking: 

- `appTimeouts.httpUpstreamTimeout`, `-app-timeouts-http-upstream-timeout`
- `appTimeouts.wessocketUpstreamTimeout`, `-app-timeouts-websocket-upstream-timeout`

Suggestion:
- `httpUpstreamTimeout`, `-http-upstream-timeout`
- `wsUpstreamTimeout`, `-ws-upstream-timeout`

Or:
- `upstream.httpTimeout`, `-upstream-http-timeout`
- `upstream.wsTimeout`, `-upstream-ws-timeout`

Or:
- `httpTimeout`, `-http-timeout`
- `wsTimeout`, `-ws-timeout`

You can use `configKey:"-"` if you want a specific struct for timeouts, but I think that the structure is not necessary.